CONTRIBUTING.md

```diff
@@ -113,8 +115,8 @@ information on using pull requests.
 
   * Sign the CLA (only needed once per user, see above).
 
-  * AUTHORS: If this is your first contribution, add your name or your
-  company name to the [AUTHORS](AUTHORS) file for copyright tracking purposes.
+  * AUTHORS: If this is your first contribution, you can add your name
```

Comment:
Even if it is not the first contribution, no?

Reply:
True, removed the condition